WORKING_GROUPS.md

```diff
@@ -99,7 +99,7 @@ The i18n working groups handle more than just translations. They
 are endpoints for community members to collaborate with each
 other in their language of choice.
 
-Each team is organized around a common spoken language. Each
+Each team is organized around a commonly spoken language. Each
```

Comment:
The idea is that a working group has a common language. Not that the group needs to be speaking a commonly spoken language